src/listbox/index.ts

```diff
@@ -36,7 +37,11 @@ const Listbox = <I>(props: Props<I>) => {
 		[itemHeight],
 	);
 
-	return html`<div class="items" ${ref((el) => (listRef.current = el))}>
+	return html`<div
+		class="items"
+		${ref((el) => (listRef.current = el))}
+		style="${styleMap({ minHeight: height + 'px' })}"
```

Comment:
I think that `styleMap` is overkill for this scenario. Let's keep it simple.

```suggestion
		style="min-height: ${height}px"
```